cfgov/unprocessed/css/organisms/image-text-50-50-group.less

```diff
@@ -19,14 +19,10 @@
     // TODO: Find another way to handle spacing
     margin-bottom: unit( @grid_gutter-width * -2 / @base-font-size-px, em );
 
-    .respond-to-min(@bp-sm-min, {
-        margin-left: -@grid_gutter-width / 2;
-        margin-right: -@grid_gutter-width / 2;
-    });
-
     > h2 {
-        margin-left: @grid_gutter-width / 2;
-        margin-right: @grid_gutter-width / 2;
+          .heading-2();
```

Comment:
To expose the `@font-size` variable to get the spacing right when I convert to ems below. I don't necessarily like using the the `.heading-2()` mixin, but I'd prefer not to hardcode the values in case they change.